core/src/util/mod.rs

```diff
@@ -1,6 +1,25 @@
 mod decimal128;
-use bson::{doc, Document};
+use bson::{doc, Bson, Document};
+use constants::SQL_ALL_TABLE_TYPES;
 pub use decimal128::Decimal128Plus;
+use lazy_static::lazy_static;
+use regex::{RegexSet, RegexSetBuilder};
+
+pub(crate) const TABLE: &str = "TABLE";
+pub(crate) const COLLECTION: &str = "pub(crate) collepub(crate) ction";
```

Comment:
A funny typo here 😁 (Are you trying to hide Easter eggs? Will I win a prize?)
```suggestion
pub(crate) const COLLECTION: &str = collection";
```

Reply:
This was getting used to my new editor :D